packages/services/f-feature-management/package.json

```
@@ -35,11 +37,15 @@
     "babel-loader": "8.1.0",
     "eslint-plugin-json-schema-validator": "^1.0.48",
     "eslint-plugin-jsonc": "^1.3.1",
+    "@vue/cli-plugin-eslint": "3.9.2",
     "jest": "25.4.0",
     "jest-teamcity": "1.7.0",
     "uglifyjs-webpack-plugin": "2.2.0",
     "webpack": "4.42.1",
     "webpack-cli": "3.3.11"
   },
-  "peerDependencies": {}
+  "peerDependencies": {},
+  "dependencies": {
+    "crypto-js": "^4.1.1"
```

Can you remove the `^` – just helps with locking versions between machines:

```suggestion
    "crypto-js": "4.1.1"
```